ProxyAppsSuite/tests/examini.yaml

```
@@ -0,0 +1,102 @@
+
+examini:
+   summary: Research vehicle for particule codes in Molecular Dynamics which allows modular independent investigation of different aspects such as force calculation, communication, neighbor list construction and binning.
+  
+
+   build:
+       modules: [ gcc, cmake, openmpi/2.1.2 ]
+       source_path: ExaMini
+       env:
+          CC: mpicc
+       cmds:
+           # Building KOKKOS
+           - git clone https://github.com/kokkos/kokkos
+           - echo "beginning kokkos build"
+           - cd kokkos
+           - git checkout 3.0.00
+           - mkdir ../kokkos-build
+           - cd ../kokkos-build
+           - cmake ../kokkos/ -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=${PWD}/../kokkos-3.0.00 -DKokkos_ENABLE_OPENMP=ON -DKokkos_ENABLE_TESTS=ON -DKokkos_ENABLE_EXAMPLES=OFF -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_CXX_EXTENSIONS=OFF -DCMAKE_BUILD_TYPE=RELEASE
+           - make -j 
+           - make install 
+           - cd ../
+           # Building ExaMiniMDi
+           -  git clone https://github.com/ECP-copa/ExaMiniMD
+           - cd ExaMiniMD/src
+           - echo "beginning ExaMiniMD build"
+           - make -j KOKKOS_PATH=../../kokkos 
+
+   scheduler: slurm
+   slurm:
+      num_nodes: 1
+      tasks_per_node: 2
+ 
+   subtitle: "{{input.test_id}}"
+   variables:
+     input:
+        - { test_id: "test1", file: "in.lj", comm_type: "MPI", threads: '12', devices: '2' }    
+   run:
+     modules: [ gcc, cmake, openmpi/2.1.2 ]
+     env:
+        OMP_PLACES: 'THREADS'
+        OMP_PROC_BIND: 'SPREAD'
+ 
+     cmds:
+         - cd ExaMiniMD/src
+         - srun ./ExaMiniMD  -il ../input/{{input.file}} --comm-type "{{input.comm_type}}" --kokkos-threads={{input.threads}} --kokkos-num-devices={{input.devices}}  &> {{input.test_id}}.out 
+         - cd ../../
+         - mv ExaMiniMD/src/{{input.test_id}}.out {{input.test_id}}.out
+         - cat {{input.test_id}}.out
+  
+   result_parse:
+      constant:
+         file_name:
+            const: '{{input.file}}'
+         comm_type:
+            const: '{{input.comm_type}}'
+         threads:
+            const: '{{input.threads}}'
+         devices:
+            const: '{{input.devices}}'
+      regex:
+         results:
```

```suggestion
         result:
```

We want it to override the `result` key
``` 
'result': 'PASS',
'results': True,
```